app/src/main/java/com/mapbox/navigation/examples/basics/ShowSpeedLimitActivity.kt

```diff
@@ -201,9 +198,14 @@ class ShowSpeedLimitActivity : AppCompatActivity() {
     /**
      * Register the observer to listen to map matching events to get the speed limit related information.
      */
-    private val mapMatcherObserver = MapMatcherResultObserver { mapMatcherResult ->
-        val value = speedLimitApi.updateSpeedLimit(mapMatcherResult.speedLimit)
-        binding.speedLimitView.render(value)
+    private val locationMatcherObserver = object : LocationObserver {
```

Comment:
Do we really need this. I think we can just add it to the same `val locationObserver` above.